WcaOnRails/app/models/competition.rb

```diff
@@ -489,7 +490,11 @@ def info_for(user)
       info[:upload_results] = I18n.t('competitions.messages.upload_results')
     end
     if self.in_progress? && !self.cancelled?
-      info[:in_progress] = I18n.t('competitions.messages.in_progress', date: I18n.l(self.end_date, format: :long))
+      if self.use_wca_live_for_scoretaking
+        info[:in_progress] = I18n.t('competitions.messages.in_progress_at_wca_live_html', link_here: self.wca_live_link?).html_safe
```

Comment:
Just curious: What happens if you omit the `.html_safe` part?
I have a vague memory of Ruby `.erb` templates handling HTML encoding automagically, but I'm really not sure so I'd like to find out 😃

Reply:
It will how as follows:
```
Note: This competition is ongoing. You can check the live results <a href='https://live.worldcubeassociation.org/link/competitions/My0ResultsComp2022'>here</a>!
```

Reply:
Thanks, makes sense to include the `html_safe` then!